src/Logger.ts

```diff
@@ -0,0 +1,31 @@
+const ESC = "\u001b"
+const RESET = ESC + "[m"
+const GRAY  = ESC + "[37m"
+const WHITE = ESC + "[97m"
+const DARK  = ESC + "[90m"
+const RED   = ESC + "[91m"
+
+export default class Logger {
+
+	constructor(private readonly actor: string) {}
+	
+	log(message: string) {
```

Comment:
Make it log(...message: string)
And then make it log all of the args, like with console.log

Reply:
That breaks the coloring
Unless I do `message.join(" ")` of course
Should I do that?

Reply:
yes

Reply:
addressed by: [2e81dde](https://github.com/creelonestudios/mailverse/pull/9/commits/2e81dde9c7433f77df3c5266410145f61422eee4)